src/cirrus/lib2/events.py

```diff
@@ -97,14 +97,14 @@ def announce(
         message = {
             "event_type": event_type,
             "payload": payload,
-            "payload_id": payload_id,
+            "id": payload_id,
```

Comment:
@jkeifer -- Two bits here:

1. We had discussed having `payload_id`, and I had moved to `id` but am not thinking that we should stick with `payload_id`.  An `id` element in the message feels more like it would be the message ID.  What do you think?
2. The initial proposed message format included `workflow_name` in the message.  I did not include it as it is part of the `payload_id` (assuming cirrus workflow_id convention `{collections}/workflow-{workflow_name}/{item_str}`).  Can you share any thoughts on the case for including it as a top level parameter?

Reply:
I never answered this, sorry...

I think `payload_id` is better than `id`, else it seems ambiguous, like we could be meaning something like "event ID".

As to including `workflow_name`, I don't remember where that came into play, honestly. If I suggested it be part of the message itself then 🤷, but I agree it doesn't need to be part of the message.

That said, I believe it really should be part of the message attributes. It seems rather natural to expect an event listener could be concerned with only the events pertaining to a specific workflow--but as we discussed, more attributes can be added later, so I suppose it is okay to leave this as-is for now.